src/JBrowse/View/Track/_FeatureDetailMixin.js

```diff
@@ -159,7 +159,7 @@ return declare( FeatureDescriptionMixin, {
                 },
                 container )
             additionalTags.sort().forEach( t => {
-                this.renderDetailField( atElement, t, f.get(t), f, undefined, track.store.getTagMetadata(t))
+                this.renderDetailField( atElement, t, Util.escapeHTML(f.get(t)), f, undefined, track.store.getTagMetadata(t))
```

Comment:
`f.get(t)` here can be an object, so escaping it is causing errors when I try to view details for a variant.

Reply:
yep...pretty obvious miss there